app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Filtering applications by subject
+description: Let users filter the applications list by the course’s subject
+date: 2021-02-25
+---
+
+Users need to be able to filter their applications by subject.
+
+## Course data analysis
+
+Providers using the Manage service offer approximately 40 subjects at primary and secondary levels.
+
+The number of subjects offered varies between providers.
+
+- Average number of subjects offered: 10
+- Maximum number of subjects offered: 28
+- Minimum number of subjects offered: 1
+
+Some courses are made up of multiple subjects, for example:
```

Comment:
I've used the correct capitalisation for courses (lower case except languages). In the service many will have an initial capital just because that's how they're displayed in the filter list.